packages/create-astro/src/index.ts

```diff
@@ -19,6 +19,8 @@ process.on('SIGTERM', exit);
 // https://github.com/withastro/docs/blob/main/src/pages/en/install/auto.md
 // if you make any changes to the flow or wording here.
 export async function main() {
+	// Clear console because PNPM startup is super ugly
+	console.clear();
```

Comment:
Does this contain important information for the user?

Reply:
In my experience no, it just logs out a stream while downloading packages. We'll only be able to clear if everything successfully downloaded and `create-astro` is running, so not a risk to clear IMO.